docs/03-assertions.md

```diff
@@ -163,53 +174,69 @@ test('custom assertion', t => {
 
 ## Built-in assertions
 
+**Note**: Unlike other assertion libraries, these assertions
+do ***not*** throw on failure.
+
 ### `.pass(message?)`
 
 Passing assertion.
 
 ### `.fail(message?)`
 
-Failing assertion.
+Failing assertion. Does not throw.
```

Comment:
I'd rather remove this. This describes how the specific assertion behaves, not sure it aids readability to repeat that the assertion never throws.